docs/pywbemcligeneraloptions.rst

```diff
@@ -61,166 +70,470 @@ command, general the options redefinition's are not retained between command
-the exact help text output shown in section :ref:`pywbemcli Help Command
-Details`):
+This section defines in detail the requirements, characteristics, and any special
+syntax of each general option
 
-* **--server/-s** - Host name or IP address of the WBEM server to which
-  pywbemcli will connect in the format::
+.. _`--server general option`:
 
-    [{scheme}://]{host}[:{port}]
+--server general option
+^^^^^^^^^^^^^^^^^^^^^^^
 
-  Where:
+The value of the ``--server/-s`` general option is a string that is the host
+name or IP address of the WBEM server to which pywbemcli will connect in the
+format::
 
-  * Scheme: must be "https" or "http" [Default: "https"].
-  * Host: defines short/fully qualified DNS hostname, literal
-    IPV4 address (dotted), or literal IPV6 address. see :term:`RFC3986` and
-    :term:`RFC6874`
-  * Port: (optional) defines WBEM server port to be used [Default: 5988(HTTP)
-    and 5989(HTTPS)]..
+    [{Scheme}://]{Host}[:{Port}]
 
-  This option is mutually exclusive with the ``--name`` option and the
-  ``--mock-server`` option since each defines a connection to a WBEM server.
+Where:
 
-  In the interactive mode this connection is not actually executed until a
-  command requiring access to the WBEM server is entered.
+* Scheme: must be "https" or "http" [Default: "https"].
+* Host: defines short/fully qualified DNS hostname, literal
+  IPV4 address (dotted), or literal IPV6 address. see :term:`RFC3986` and
+  :term:`RFC6874`
+* Port: (optional) defines WBEM server port to be used [Default: 5988(HTTP)
+  and 5989(HTTPS)]..
 
-  Examples for the `URL` parameter of this option:
+This option is mutually exclusive with the :ref:`--name general option` and the
+:ref:`--mock-server general option` since each defines a connection to a WBEM server.
 
-  .. code-block:: text
+In the interactive mode the connection is not actually executed until a
+command requiring access to the WBEM server is entered.
 
-      https://localhost:15345 (http port 5989, host name localhost)
-      http://10.2.3.9 (port 5988, ipv4 ip address 10.2.3.9)
-      http://[2001:db8::1234-eth0] -(http port 5988 ipv6, zone id eth0)
+Examples for the `URL` parameter of this option include:
 
-* **--name/-n** - The name of a WBEM server that is defined in the
-  :term:`connections file`.  The server parameters for this connection name will
-  be loaded from the :term:`connections file` to become the current WBEM
-  connection in pywbemcli. Note: In the interactive mode a connection is not
-  actually used until a command requiring access to the WBEM server is entered.
-  This option is mutually exclusive with ``--server`` and ``--mock-server``
-  since each option defines a WBEM server for pywbemcli.
+.. code-block:: text
 
-  A new WBEM server named (``myserver``) may be defined and saved in the connection
-  file with command as follows:
+  https://localhost:15345 (http port 5989, host name localhost)
```

Comment:
* in the parenthesis, change "http" to "https"
* in the parenthesis, add a comma after "https"

Reply:
DONE  did you mean add coma after ipv4 and after http port 5988 in third example?

```

      https://localhost:15345 (https port 5989, host name localhost)
     http://10.2.3.9 (http port 5988, ipv4, ip address 10.2.3.9)
     http://[2001:db8::1234-eth0] (http port 5988, ipv6, zone id eth0)

Reply:
* In the example you show above, remove the comma after IPv4 in second example.
* I suggested adding commas after http and https, because that is the protocol, and while it is true that there are specific ports for each protocol, that is really independent.
* Lastly: First example specifies port 15345 so in the parenthesis change 5989 to 15345.

Reply:
DONE